Pet-Clinic-Frontend/src/pages/Login.tsx

```diff
@@ -36,9 +38,10 @@ const Login = () => {
       });
       if (!response.status.toString().startsWith('2')) {
```

Comment:
```suggestion
      if (!response.status.toString().startsWith('2')) {
```
I think we should not make this number a string to check if it is 2xx, we can check it with numbers too.

Reply:
If i use for example (response.status < 200 || response.status >= 300).  Why is better, it is better?

Reply:
Well status codes are numbers and we should treat them as such, I am not saying this does not work, but it is debatable, which one is more understandable.